understanding/20/headings-and-labels.html

```diff
@@ -30,6 +30,9 @@ <h2>Intent of Headings and Labels</h2>
       <p>Labels and headings do not need to be lengthy. A word, or even a single character,
          may suffice if it provides an appropriate cue to finding and navigating content.
       </p>
+       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. In these cases, authors would need to ensure that the image and its use as label in a particular context can be considered conventional and widely understood. 
```

Comment:
```suggestion
       <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. In these cases, authors should ideally ensure that the image and its use as a label (in context) are widely understood. 
```

Reply:
I don't think "descriptive" from the normative text means that it has to be considered conventional or widely understood, so I'd frame that as a useful thing to do but not required.